### ISSUE TITLE
For `Progress indicator` component, error and warning status should be read by Voice Over

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1697]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 17

### Device OS version

iOS 26.5.1

### Is Liquid Glass disabled

- [ ] Yes, Liquid Glass is disabled (or OS version is older than 26)
- [x] No, Liquid Glass is enabled (or OS version is 26 or newer)

### OUDS iOS library version

3.0.0

### Describe the issue

Negative and warning status are not read by Voice Over

### Expected Behavior

At least negative (error) and warning status should be read by Voice Over.

Maybe let the possibility to add a description for others status.

### Device settings

Voice Over

**Comment by pylapp** [comment]
@pya35  Ces composants ne sont pas censés être utilsié seuls, et peuvent avoir un _helper text_ (oublié pour le _circular proress indicator_ mais il va venir).

Queelle serait la meilleure façon de faire dans ce cas là ?

**Comment by pya35** [comment]
> [@pya35](https://github.com/pya35) Ces composants ne sont pas censés être utilsié seuls, et peuvent avoir un _helper text_ (oublié pour le _circular proress indicator_ mais il va venir).
> 
> Queelle serait la meilleure façon de faire dans ce cas là ?

Je pense que c'est préférable de remonter le statut tout de même pour que les états warning/error soient bien identifiés  au cas où le helper text n'est pas présent ou pas suffisamment explicite. Un peu comme le cas d'erreur sur le `text input`

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **3.0.0**
- Build number: **1788815142**
- Build tag: **d979f56**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **3.0.0**
- Build number: **1788989831**
- Build tag: **5e2d2a8**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **3.0.0**
- Build number: **1789074302**
- Build tag: **bb28c6d**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New release available** 🚀

Please, get the [new release from GitHub](https://github.com/Orange-OpenSource/ouds-ios-design-system-toolbox/releases/tag/3.0.0) and wait for App Store update to test it!

- Display name: **Design System Toolbox**
- Version: **3.0.0**
- Build number: **1789078720**



_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
